infrastructure/modules/catcolab/services.nix

```diff
@@ -135,17 +143,31 @@ with lib;
 
       serviceConfig = {
         User = "catcolab";
-        Type = "notify";
+        Type = "simple";
         Restart = "on-failure";
         ExecStart = lib.getExe backendPkg;
         EnvironmentFile = cfg.environmentFile;
       };
     };
 
+    # There is a possible problem: this service could be starting up WHILE migrations on the automerge
+    # docs are being run for the backend startup. I don't think this will cause any issues. The alternative is
+    # moving migrations out of backend (which causes many more problems), or have the backend process control
+    # the automerge process instead of systemd.
     systemd.services.automerge = {
       enable = true;
       wantedBy = [ "multi-user.target" ];
 
+      after = [
+        "database-setup.service"
+        "network-online.target"
+      ];
```

Comment:
This missing block is what was causing me grief.

Reply:
Good find. I guess we were just getting lucky with the timing of the services starting up.